src/test/resources/com/puppycrawl/tools/checkstyle/checks/InputAvoidEscapedUnicodeCharacters.java

```diff
@@ -102,4 +102,17 @@ void foo() {
 	private String notAUnicodeEscaped2 = "\\\\u1234";
 
 	private String onlyEscaped = "\\\u1234";
+
+	private String controlC0 = ""
+			+ "\u0000\u0001\u0002\u0003\u0004\u0005\u0006\u0007\u0008\u0009\n\u000b\u000c\r\u000e\u000f"
```

Comment:
We need clear identification of which ones are good and bad. Violation just shows the line number and not the column number.
It is not clear if all are bad or just one is bad and the others are accidentally good. It won't be clear if we change expression in the future and mess up one of the characters.
Input should also have a comment identifying which are good or bad as a secondary verification.

@romani Please confirm the range of numbers is ok.
Should we expand this to every number between 0000 and FFFF?